### ISSUE TITLE
g-movements not working in non-latin layouts

### ISSUE THREAD

**Issue by yerlaser** [71/dance#123]
It has probably been reported before, but I couldn't find it. Feel free to mark it a duplicate in one exists.

When using non-latin layout majority of the commands work as if using US QWERTY which is a good and welcoming feature.
This doesn't work however for movements like gh, gl, etc.
I assume that's because only the first letter (in this case g) is mapped as a layout independant shortcut and the second is read as a normal symbol. Since the symbol is non-latin it doesn't work.
Possible solution would be to map all two letter mappings gg, gl, gh, etc.
This, however, will prevent showing the hints.

A better solution would be something similar to keymaps in VIM (:help keymap) which allows one to stay in US QWERTY while in insert mode (and, e.g. searching) translate characters to local symbols.

**Comment by yerlaser** [comment]
Hey Man, thanks a lot!
At first I didn't understand what you mean and started to search through the source code to find anything resembling dance.menus file/variable/class etc.
:)

Only after a while I figured that there exists a user facing setting in the settings file.
I managed to configure it according to my keyboard layout.

For those who face this same problem, keep in mind that when you click on the setting it creates a copy of default menu object.
Don't add your values as it creates a duplicate, instead replace the characters with the ones in your layout.
